### DISCUSSION TITLE
How critical to use .NET6 but not a newer version?

### DISCUSSION THREAD

**Discussion by dkurt** [actions/runner#3098] [Ideas] [answered]
Runner uses .NET6 however modern CPU architectures have better support on newer .NET versions. I'm [working on RISC-V runner](https://github.com/dkurt/github_actions_riscv), there are also requests for [Loongson](https://github.com/actions/runner/issues/2157#issuecomment-1809463234), [ppc64le ](https://github.com/actions/runner/pull/2386), [s390x](https://github.com/actions/runner/issues/2263).

**Comment by dkurt** [comment]
proposal: https://github.com/actions/runner/pull/3099

**Comment by nebuk89** [comment, marked as answer]
https://github.blog/changelog/2024-08-19-notice-of-upcoming-deprecations-and-breaking-changes-in-github-actions-runners/ 

We are moving to .NET8 :)
